### PR TITLE
Apps integration

### PR DESCRIPTION
**Pull request by multi-Adams**
added section three of app-integrations page

### CODE REVIEW

File: control/src/pages/contact-us/components/ContactMain.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import contactStyle from '../styles/contactMain.module.css'
-import ContactUs from './ContactUsImg.svg'
+import ContactUs from './ContactUsImg.png'
```
Comment (ickynavigator) [inline control/src/pages/contact-us/components/ContactMain.js:2]: convert to svg

File: control/src/pages/contact-us/components/contactFormContainer/contactFormContainer.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import detailsData from './detailsArray'
 import axios from 'axios'
 import { useDropzone } from 'react-dropzone'
 import Alert from '../../assets/alert-circle.svg'
+import { GetUserInfo } from '../../../../zuri-control'
```
Comment (ickynavigator) [inline control/src/pages/contact-us/components/contactFormContainer/contactFormContainer.js:7]: @pidoxy is this the right import or "@zuri/control"?

**Review by ickynavigator — CHANGES_REQUESTED** [review]

File: control/src/pages/contact-us/components/contactFormContainer/contactFormContainer.js (hunk 2 above)
Comment (pidoxy) [reply control/src/pages/contact-us/components/contactFormContainer/contactFormContainer.js:7]: This is correct as well since it's in the same repo

**Review by multi-Adams — COMMENTED** [review]
review done

File: control/src/pages/contact-us/components/contactFormContainer/contactFormContainer.js (hunk 2 above)
Comment (ickynavigator) [reply control/src/pages/contact-us/components/contactFormContainer/contactFormContainer.js:7]: alright

**Review by ickynavigator — APPROVED** [review]

**Pull request merged**